code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -260,3 +260,75 @@
 /obj/item/weapon/invisible_spray/permanent
 	desc = "A can of... invisibility?"
 	permanent = 1
+	
+/obj/item/weapon/razor
+	name = "electric razor"
+	desc = "The latest and greatest power razor born from the science of shaving."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "razor"
+	w_class = W_CLASS_TINY
+
+
+/obj/item/weapon/razor/proc/shave(mob/living/carbon/human/H, location = "mouth")
+	if(location == "mouth")
+		H.f_style = "Shaved"
+	else
+		H.h_style = "Skinhead"
+
+	H.update_hair()
+	playsound(loc, 'sound/items/Welder2.ogg', 20, 1)
+
+
+/obj/item/weapon/razor/attack(mob/M, mob/user)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/location = user.zone_sel.selecting
+		if(location == "mouth")
+			if(H.check_body_part_coverage(MOUTH))
+				to_chat(user,"<span class='warning'>The mask is in the way!</span>")
+				return
+			if(H.f_style == "Shaved")
+				to_chat(user,"<span class='warning'>Already clean-shaven!</span>")
+				return
+
+			if(H == user) //shaving yourself
+				user.visible_message("[user] starts to shave their facial hair with [src].", \
+									 "<span class='notice'>You take a moment to shave your facial hair with [src]...</span>")
+				if(do_after(user, H, 5))
+					user.visible_message("[user] shaves his facial hair clean with [src].", \
```

Comment:
>his

YOU DON'T KNOW ME

And before you ask, yes, Character Setup lets females have facial hair.